go/http/httpbase.go

```diff
@@ -27,20 +28,39 @@ import (
 	"github.com/github/orchestrator/go/inst"
 	"github.com/github/orchestrator/go/os"
 	"github.com/github/orchestrator/go/process"
-	"github.com/github/orchestrator/go/raft"
+	orcraft "github.com/github/orchestrator/go/raft"
```

Comment:
is `orcraft ` expected here?

Reply:
> is `orcraft ` expected here?

John, it is an alias and distinguishes the `orchestrtor` raft module from any other. It is then used when referencing variables and calling functions etc. This is not a change from this PR, but exists in the main branch.

Reply:
Also, this PR will not be merged and remains as a reminder for @shlomi-noach for token-controlled authentication